examples/loops-param-result-array.yaml

```diff
@@ -0,0 +1,43 @@
+# This workflow demonstrates the use of a generator step which produces a list within list of items as a result.
```

Comment:
Can we make this a test and not an example. Don't think it's necessary to make as an example